templates/ingress.yaml

```diff
@@ -23,6 +23,14 @@ metadata:
     nginx.ingress.kubernetes.io/auth-url: https://houston.{{ .Values.ingress.baseDomain }}/v1/authorization
     nginx.ingress.kubernetes.io/auth-signin: https://app.{{ .Values.ingress.baseDomain }}/login
     nginx.ingress.kubernetes.io/auth-response-headers: authorization, username, email
+    nginx.ingress.kubernetes.io/proxy-cookie-path: / /{{ .Release.Name }}/
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      if ($host = '{{- include "old_airflow_url" . }}' ) {
+        rewrite ^ https://deployments.{{ .Values.ingress.baseDomain }}/{{ .Release.Name }}/airflow permanent;
```

Comment:
@andriisoldatenko could these snippets be shortened to use your new `deployments_host` helper?

Reply:
fixed.